README.md

```diff
@@ -141,6 +141,62 @@ const searchResult = await shopperSearch.productSearch({
 
 Invalid query parameters that are not a part of the API and do not follow the `c_` custom query parameter convention will be filtered from the request and a warning will be displayed.
 
+### Custom APIs
+
+The SDK supports calling custom APIs with a helper function, `callCustomEndpoint`.
+
+Example usage:
+
+```javascript
+import pkg from 'commerce-sdk-isomorphic';
+const { helpers } = pkg;
+
+const CLIENT_ID = "<your-client-id>";
+const ORG_ID = "<your-org-id>";
+const SHORT_CODE = "<your-short-code>";
+const SITE_ID = "<your-site-id>";
+
+// client configuration parameters
+const clientConfig = {
+  parameters: {
+    clientId: CLIENT_ID,
+    organizationId: ORG_ID,
+    shortCode: SHORT_CODE,
+    siteId: SITE_ID,
+    // Custom API path parameters
+    endpointPath: 'customers',
+    apiName: 'loyalty-info',
+    apiVersion: 'v1', // defaults to v1 if not provided
+  },
+};
+
+// Flag to retrieve raw response or data from helper function
+const rawResponse = false;
+const accessToken = '<INSERT ACCESS TOKEN HERE>';
+
+let response = await helpers.callCustomEndpoint(
+  {
+    method: 'GET',
+    parameters: {
+      queryParameter: 'queryParameter1',
+      siteId: SITE_ID,
+    },
+    headers: {
+      'Content-Type': 'application/json',
```

Comment:
Do we expect callers to pass the content type or can it be automatically inferred or default to JSON?

Reply:
Updated implementation to default to `"Content-Type" : "application/json"` if `Content-Type` header is not provided